docs/boards/get-started/customize-boards-agile.md

```diff
@@ -25,7 +25,7 @@ This article shows how to customize a Kanban board for the Agile process. You ha
 > [!IMPORTANT]  
 > **Select the version that meets your location and process**:
 > We are experimenting with a new acquisition model which is 
-> currently available for users located in the United States and that sign up through [azure.com/boards](https://azure.microsoft.com/en-us/services/devops/boards/?nav=min). This model supports a new Basic process.  
+> currently available for users located in the United States and that sign up through [azure.com/boards](https://azure.microsoft.com/services/devops/boards/?nav=min). This model supports a new Basic process.  
```

Comment:
This might be intentional because it says it's only US. The azure.com/boards takes you to the local redirect site though